internal/query/projection/web_key.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 const (
-	WebKeyTable = "projections.web_keys"
+	WebKeyTable = "projections.web_keys1"
```

Comment:
We don't have to bump as long as we release it together with the commit that created the projection, right?

Reply:
As discussed on internal chat, `main` branch with `projections.web_keys` is already deployed on QA. It's easier to just bump the version instead of manual cleanup.